dashboard/db/schema.rb

```diff
@@ -742,6 +742,8 @@
     t.integer  "script_id",       limit: 4
     t.integer  "level_source_id", limit: 4
     t.boolean  "submitted"
+    t.boolean  "view_answers"
```

Comment:
I remember it was discussed, but from this name I can't quite tell what this field means.  Is it a capability?

For example, with submitted I know it means that this activity has been submitted.

I'm not sure what the same sentence is that uses view_answers.